source/shared/net/mobile-wallet/shield-unshield-mw.rst

```diff
@@ -24,17 +24,21 @@ Move CCD to the shielded balance
 
 #. Go to the **Accounts** page.
 
-#. Tap the account you want to transfer CCD from, and tap the **Balance** area of the account card.
+#. Tap the **Balance** area of the account you want to transfer CCD from or tap |moredetails|.
 
-#. Tap **SHIELD**.
+#. If the **Shield** button and **Shielded balanace** tab are not visible, tap the |hamburger| menu in the upper right corner.
+
+#. Tap **Show shielded balance on account name**. Now the Account card contains tabs for both **Balance** and **Shielded Balance** and a **Shield** button.
```

Comment:
```suggestion
#. Tap **Show shielded balance on account name**. Now the totals area contains tabs for both **Balance** and **Shielded Balance** and a **Shield** button.
```

Accunt card usually refers to the cards on the account overview page, where you can see all your accounts. I'm not sure if "totals area" is better though.